packages/amplify_auth_cognito/ios/Classes/FlutterConfirmSignInRequest.swift

```diff
@@ -29,8 +29,16 @@ struct FlutterConfirmSignInRequest {
   }
     
   func formatOptions(options: Dictionary<String, Any>?) -> AuthConfirmSignInOperation.Request.Options {
-     
+    let rawAttributes = options?["userAttributes"] as? [String : String]
+    var userAttributes: [AuthUserAttribute] = [];
+    if (rawAttributes != nil) {
+      for (key, value) in rawAttributes ?? [:] {
+        userAttributes.append(AuthUserAttribute(createAuthUserAttributeKey(keyName: key), value: value))
+      }
+    }
+    
```

Comment:
Similar to kotlin counterpart
```suggestion
    var rawAttributes = options?["userAttributes"] as? [String : String] ?? [:]
    var userAttributes: [AuthUserAttribute] = rawAttributes.map { key, value in
      AuthUserAttribute(createAuthUserAttributeKey(keyName: key), value: value)
    }
```

Even if we don't take this suggestion, I think we should move the `?? [:]` from line 35 to line 32. I think that might make it possible to have `rawAttributes` be a `var` instead of `let` as well?

Reply:
Done